analyses/WI_cd_2000/01_prep_2000_WI_cd_2000.R

```diff
@@ -0,0 +1,80 @@
+###############################################################################
+# Download and prepare data for `WI_cd_2000` analysis
+# © ALARM Project, August 2025
+###############################################################################
+
+suppressMessages({
+    library(dplyr)
+    library(readr)
+    library(sf)
+    library(redist)
+    library(geomander)
+    library(baf)
+    library(cli)
+    library(here)
+    devtools::load_all() # load utilities
+})
+
+# Download necessary files for analysis -----
+cli_process_start("Downloading files for {.pkg WI_cd_2000}")
+
+path_data <- download_redistricting_file("WI", "data-raw/WI", year = 2000, overwrite = TRUE)
+
+cli_process_done()
+
+# Compile raw data into a final shapefile for analysis -----
+shp_path <- "data-out/WI_2000/shp_vtd.rds"
+perim_path <- "data-out/WI_2000/perim.rds"
+
+if (!file.exists(here(shp_path))) {
+    cli_process_start("Preparing {.strong WI} shapefile")
+    # read in redistricting data
+    df <- read_csv(here(path_data), col_types = cols(GEOID = "c"))
+
+    # join the data
+    wi_shp <- read_sf(here("data-raw/WI/wi_2000_tracts.geojson")) %>%
```

Comment:
Looks like the CB website is working again. Can you bring back the old code, as this isn't going to be available to downstream replicators.